desktop/apps/auction/components/layout/__tests__/Banner.test.js

```diff
@@ -1,10 +1,25 @@
 import renderTestComponent from 'desktop/apps/auction/__tests__/utils/renderTestComponent'
 import { test } from 'desktop/apps/auction/components/layout/Banner'
-
+import sinon from 'sinon'
 const { Banner } = test
 
 describe('auction/components/layout/Banner.test', () => {
   describe('<Banner />', () => {
+    it('tracks a click on the "Enter Live Auction" button', () => {
+      const liveAuctionUrl = 'live.artsy.net/some-auction-url'
+      const mockTrack = sinon.spy()
+      window.analytics = { track: mockTrack }
+      const { wrapper } = renderTestComponent({
+        Component: Banner,
+        props: {
+          isLiveOpen: true,
+          liveAuctionUrl
+        }
+      })
+
+      wrapper.find('a').simulate('click')
+      mockTrack.calledWithMatch('click').should.be.true()
```

Comment:
I feel like the only real logic here (aside from 'do react components and work as expected' is the formation of the `destination_path` - should I check that?)